File: wikipedia/src/main/java/org/wikipedia/settings/DeveloperSettingsActivity.java
package org.wikipedia.settings;

import android.annotation.TargetApi;
import android.content.Context;
import android.content.Intent;
import android.os.Build;

import org.wikipedia.activity.PlatformSingleFragmentActivity;

@TargetApi(Build.VERSION_CODES.HONEYCOMB)
public class DeveloperSettingsActivity extends PlatformSingleFragmentActivity<DeveloperSettingsFragment> {
    public static Intent newIntent(Context context) {
        return new Intent(context, DeveloperSettingsActivity.class);
    }

    @Override
    public DeveloperSettingsFragment createFragment() {
        return DeveloperSettingsFragment.newInstance();
    }
}

File: app/src/main/java/org/wikipedia/settings/dev/DeveloperSettingsActivity.kt
package org.wikipedia.settings.dev

import android.content.Context
import android.content.Intent
import org.wikipedia.R
import org.wikipedia.settings.BaseSettingsActivity

class DeveloperSettingsActivity : BaseSettingsActivity<DeveloperSettingsFragment>() {
    override val title = R.string.developer_settings_activity_title

    public override fun createFragment() = DeveloperSettingsFragment.newInstance()

    companion object {
        fun newIntent(context: Context) = Intent(context, DeveloperSettingsActivity::class.java)
    }
}
